### PR TITLE
Feature/new gh filter

### PR DESCRIPTION
**Pull request by jamdrop**
allows e.g.

```
ls org:acme review-requested:acme/dev-team
```

### CODE REVIEW

**Comment by ohbarye** [comment]
@jamdrop 
Thanks for the PR. But I assume that the bot already provides those filters like below.

- You can use `owner` instead of `org`.
- You can use `reviewer` instead of `review-requested`.

Are they insufficient?

**Comment by jamdrop** [comment]
the reviewer is only applied after all PRs are loaded as i understand the code, thats bad for large orgs

and the user syntax somehow does not work

i tried
```
ls user:acme author:foo
```
and get PRs outside acme (which shouldnt be)

**Comment by ohbarye** [comment]
Hmm, I tried quite some times so that I could reproduce your situation. But couldn't.
Once I can do that, I'm going to review this PR. Untile then, please use your forked version if you wouldn't mind.

File: src/GitHubApiClient.js [hunk 1]
```diff
@@ -152,15 +152,31 @@ class GitHubApiClient {
...
         return this.isTeam(author) ? this.getTeamMembers(author) : author
       })
     )).flatten()
       .uniq()
       .value()
-    const prs = await this.getPullRequestsForAuthors(author, repo, user)
+
+    reviewRequested.values = _(await Promise.all(
+      reviewRequested.values.map((reviewer)=> {
+        return this.isTeam(reviewer) ? this.getTeamMembers(reviewer) : reviewer
```
Comment (sgrodzicki) [inline src/GitHubApiClient.js]: For PRs with requests for a specified team [GitHub offers](https://github.blog/2017-08-24-easily-filter-review-requests-by-team/) a `team-review-requested:TEAMNAME` qualifier

**Pull request closed without merging**